content/en/docs/concepts/security/linux-kernel-security-constraints.md

```diff
@@ -0,0 +1,270 @@
+---
+title: Linux kernel security constraints for Pods and containers
+description: >
+  Overview of Linux kernel security modules and constraints that you can use to
+  harden your Pods and containers.
+content_type: concept
+weight: 100
+---
+
+<!-- overview -->
+
+This page describes some of the security features that are built into the Linux
+kernel that you can use in your Kubernetes workloads. To learn how to apply
+these features to your Pods and containers, refer to
+[Configure a Security Context for a Pod or Container](/docs/tasks/configure-pod-container/security-context/).
+You should already be familiar with Linux and with the basics of Kubernetes
+workloads.
+
+<!-- body -->
+
+## Run workloads without root privileges {#run-without-root}
+
+When you deploy a workload in Kubernetes, use the Pod specification to restrict
+that workload from running as the root user on the node. You can use the Pod
+securityContext to define the specific Linux user and group for the processes in
+the Pod, and explicitly restrict containers from running as root users. Setting
+these values in the Pod manifest takes precedence over similar values in the
+container image, which is especially useful if you're running images that you
+don't own.
```

Comment:
It is worth pointing out that changing the user being used to run an application could have consequences. Changing the user/group could restrain the application from accessing files or being able to do some operations (e.g. listen to a privileged port - under 1024 by default). Therefore, this needs to be done with caution.